napari/_qt/_tests/test_app.py

```diff
@@ -5,7 +5,20 @@
 import pytest
 from qtpy.QtWidgets import QAction, QShortcut
 
-from napari._qt.qt_event_loop import _ipython_has_eventloop, run, set_app_id
+from napari._qt.qt_event_loop import (
+    _ipython_has_eventloop,
+    get_app,
+    get_qapp,
+    run,
+    set_app_id,
+)
+
+
+def test_qapp(qapp):
+    qapp = get_qapp()
+    with pytest.warns(FutureWarning):
+        deprecated_qapp = get_app()
+    assert qapp == deprecated_qapp
 
 
 @pytest.mark.skipif(os.name != 'Windows', reason='Windows specific')
```

Comment:
Just in case, while checking the test addition over this file, I noticed this skip, shouldn't it be something like:

```python
@pytest.mark.skipif(os.name != 'nt', reason='Windows specific')
```

?

As it is, the test is skipped on Windows. However, if you do the skip definition change above the test is run but it fails on Windows. Not related to the changes here but maybe worthy of an issue 🤔?

Reply:
Created #7278 to follow up this :+1: